options/options.c

```diff
@@ -115,6 +115,9 @@ static const m_option_t mp_vo_opt_list[] = {
     {"ontop-level", OPT_CHOICE(ontop_level, {"window", -1}, {"system", -2},
         {"desktop", -3}), M_RANGE(0, INT_MAX)},
     {"border", OPT_BOOL(border)},
+#if HAVE_WIN32_DESKTOP
+    {"window-affinity", OPT_CHOICE(window_affinity, {"default", WDA_NONE}, {"excludefromcapture", WDA_EXCLUDEFROMCAPTURE}, {"monitor", WDA_MONITOR})},
```

Comment:
```suggestion
    {"window-affinity", OPT_CHOICE(window_affinity, {"default", WDA_NONE},
        {"excludefromcapture", WDA_EXCLUDEFROMCAPTURE}, {"monitor", WDA_MONITOR})},
```
Boring style nit, but this is a pretty long line.